specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2021-02-01/imds.json

```diff
@@ -578,6 +578,16 @@
             "description": "This contains information about the data disk.",
             "$ref": "#/definitions/DataDisk"
           }
+        },
+        "resourceDisk": {
+          "type": "object",
+          "description": "This contains data for the size of local temp disk of the VM, if it exists.",
+          "properties": {
+            "size": {
+              "type": "string",
```

Comment:
Why isn't this an integer?

Reply:
We return all numbers as strings representations.